src/sentry/auth/partnership_configs.py

```diff
@@ -3,6 +3,7 @@
 
 class ChannelName(Enum):
     FLY_IO = "fly"
+    FLY_DEACTIVATED = "fly-deactivated"
 
 
 SPONSOR_OAUTH_NAME = {ChannelName.FLY_IO: "Fly.io"}
```

Comment:
could we add something here for `FLY_DEACTIVATED` so that the second SSO class has a different name?